mysql/datadog_checks/mysql/statement_samples.py

```diff
@@ -863,21 +873,62 @@ def _run_explain(self, cursor, statement, obfuscated_statement):
         )
         return cursor.fetchone()[0]
 
-    def _run_explain_procedure(self, cursor, statement, obfuscated_statement):
+    def _run_explain_procedure(self, schema, cursor, statement, obfuscated_statement):
         """
         Run the explain by calling the stored procedure if available.
         """
-        self._cursor_run(cursor, 'CALL {}(%s)'.format(self._explain_procedure), statement, obfuscated_statement)
-        return cursor.fetchone()[0]
+        try:
+            self._cursor_run(cursor, 'CALL {}(%s)'.format(self._explain_procedure), statement, obfuscated_statement)
+            return cursor.fetchone()[0]
+        except pymysql.err.DatabaseError as e:
+            if e.args[0] in PYMYSQL_MISSING_EXPLAIN_STATEMENT_PROC_ERRORS:
+                err_msg = e.args[1] if len(e.args) > 1 else ''
+                self._log.warning("Recording warning for code explain_plan_procedure_missing")
```

Comment:
Is this leftover from debugging?

We're already logging all of these exceptions at the debug level here so I don't think we need any more logging per occurrence https://github.com/DataDog/integrations-core/blob/bade5847264e4c119fffb8150e8c9596533c6075/mysql/datadog_checks/mysql/statement_samples.py#L845

Reply:
Oops, definitely leftover from when I was debugging the state being persisted between test cases.